site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -24,10 +25,12 @@ object OAuth2 {
   }
 }
 
-class OAuth2Controller(ws: WSClient, userService: UserServices) extends Controller {
+class OAuth2Controller(implicit userService: UserServices) extends Controller {
 
   import OAuth2._
 
+  val ws = NingWSClient()
```

Comment:
We need to look into the NingWSClient lifecycle in case it needs to be closed to avoid leaks like some clients have in the past.